### ISSUE TITLE
Feature suggestion

### ISSUE THREAD

**Issue by Jukez17** [adityadroid/Messio#38]
Hi, I was thinking and it might be a nice idea that instead of automatically adding in contact in both ends the u should get an accept or ignore on upcoming contact request to avoid random people adding you.

**Comment by adityadroid** [comment]
> Hi, I was thinking and it might be a nice idea that instead of automatically adding in contact in both ends the u should get an accept or ignore on upcoming contact request to avoid random people adding you.

Yes. This was on my mind but for the sake of simplicity I didn't do it. Will probably add it later.

**Comment by Jukez17** [comment]
> > Hi, I was thinking and it might be a nice idea that instead of automatically adding in contact in both ends the u should get an accept or ignore on upcoming contact request to avoid random people adding you.
> 
> Yes. This was on my mind but for the sake of simplicity I didn't do it. Will probably add it later.

If not that atleast ability to remove contacts. Both of this I think would be great features to have.

**Comment by adityadroid** [comment]
> > > Hi, I was thinking and it might be a nice idea that instead of automatically adding in contact in both ends the u should get an accept or ignore on upcoming contact request to avoid random people adding you.
> > 
> > 
> > Yes. This was on my mind but for the sake of simplicity I didn't do it. Will probably add it later.
> 
> If not that atleast ability to remove contacts. Both of this I think would be great features to have.

Yes. That would be a good thing. I've noted it down for the next phase. Thanks for your feedback.
